Diagnostics/ExchangeLogCollector/Write/Write-LargeDataObjectsOnMachine.ps1

```diff
@@ -381,12 +380,9 @@ Function Write-LargeDataObjectsOnMachine {
             Write-Verbose("Creating Script Block")
             $getExchangeInstallDirectoryScriptBlock = [scriptblock]::Create($getExchangeInstallDirectoryString)
 
-            Write-Verbose("Getting New-Folder string to create Script Block")
-            $newFolderString = Add-ScriptBlockInjection @scriptBlockInjectParams `
-                -PrimaryScriptBlock ${Function:New-Folder} `
-                -CatchActionFunction ${Function:Invoke-CatchBlockActions}
-            Write-Verbose("Creating script block")
-            $newFolderScriptBlock = [scriptblock]::Create($newFolderString)
+            Write-Verbose("New-Item create Script Block")
+            # need to have a blank param to be able to pass an array of 2 folders.
```

Comment:
For remote execution at the same time within `Start-JobManager`. 

The paths that we need to create are something like this: 

![image](https://user-images.githubusercontent.com/22776718/160924977-2f3c8933-b460-4ee5-bead-d7caa7a19bdb.png)

`Start-JobManager` is able to kick off multiple jobs against each server and have unique arguments being passed to each server.

Reply:
I still don't follow. If you want to pass multiple paths:

![image](https://user-images.githubusercontent.com/4518572/160929225-b162f138-dbe7-41c0-bcd3-5c6a1ccab471.png)

The blank parameter is not needed.

Reply:
After testing it out, I remove the `$blank` parameter and it does still work but do need to have a fake secondary parameter to pass to the caller. Similar to what you have in the snapshot above. Otherwise, Invoke-Command doesn't know that the array paths need to be going to the 1st parameter. Instead, it would do 1 path to the first parameter, then the second path to the second parameter which doesn't exist and only one path is created.